### PR TITLE
Edits for step-70.cc

### PR DESCRIPTION
**Pull request by bangerth**
I've got to do a couple of other things for the next hour, but thought I'd already make this available to review. There is nothing here that changes functionality. It's really just edits, plus re-sorting the member variables of the class so that things that work together follow each other.

@luca-heltai @blaisb @gassmoeller: It's been my experience that it reads to rather difficult to read tutorial programs if one slices the code too thinly: Documenting individual lines and member variables makes it difficult to follow the flow of the program because one loses track of the bigger picture. So I've often combined blocks of documentation and code into larger chunks to make it easier on the eye. But there are no functional changes to all of this.

More to come later today.

/rebuild

### CODE REVIEW

File: examples/step-70/step-70.cc [hunk 1]
```diff
@@ -532,68 +567,86 @@ namespace Step70
...
+    // particles that ended up in my subdomain.
     //
     // While this is not the most efficient strategy, we keep it this way to
-    // illustrate how things would work in a real FSI problem. If a particle
-    // is linked to a specific solid degree of freedom, we are not free to
-    // choose who owns it, and we have to communicate this information around.
-    // We illustrate this here, and show that the communication pattern is
-    // point-to-point, and negligible in terms of total cost of the algorithm.
+    // illustrate how things would work in a real fluid-structure
+    // intraction(FSI_ problem. If a particle is linked to a specific solid
```
Comment (luca-heltai) [inline examples/step-70/step-70.cc:599]:
```suggestion
    // intraction (FSI) problem. If a particle is linked to a specific solid
```

Comment (luca-heltai) [reply examples/step-70/step-70.cc:599]: Only this minor typo. For the rest: thanks  @bangerth for taking the time to do this!

:)

Comment (kronbichler) [reply examples/step-70/step-70.cc:599]: Wait, shouldn't this be "interaction" rather than "intraction"?

Comment (luca-heltai) [reply examples/step-70/step-70.cc:599]: Yes!

Comment (bangerth) [reply examples/step-70/step-70.cc:599]: Yes, I had to laugh as well :-)

**Comment by bangerth** [comment]
I pushed that one line of change. Of course, this destroys the status of all 11 checks succeeding. If someone wants to be bold, wait for the mandatory checks and merge before it all runs through again :-)

**Comment by bangerth** [comment]
I've gone through the rest of the code. I kept actual code changes separate from documentation changes in the commits to make it easier to understand what I'm changing.

I think this is in great shape. We just have to address #10243 (in a separate PR).

**Pull request merged**